### ISSUE TITLE
Part 2 - Route params

### ISSUE THREAD

**Issue by luis-carlos** [ember-learn/super-rentals#156]
Hi all,

After following instructions on how to fix the error reported on thi section, test remains without passing

https://guides.emberjs.com/release/tutorial/part-2/route-params/#toc_component-tests-with-access-to-the-router

I even copied file tests/integration/components/rental-test.js from this repo and the problem persists

![image](https://user-images.githubusercontent.com/4765387/108592694-b52c3900-7334-11eb-91c7-1cdb8650d22d.png)

Are the instructions on the tutorial complete?

**Comment by fdrobidoux** [comment]
Having the same issue. Gonna have to skip those tests for now I guess. 🤷‍♂️

**Comment by ijlee2** [comment]
@luis-carlos @fdrobidoux Apologies for a late response.

I briefly looked at the tutorial page, and it seemed like the assertion `Element article h3 a has attribute "href" with value "/rentals/grand-old-mansion"/` was _meant_ to fail.

The paragraph, starting with `Well, it's about time that we ran into something that didn't Just Work™ on the first try! This is the advanced part of the tutorial after all.`, discusses how we can fix this issue.

To confirm, were you both seeing that the assertion _still_ failed after you applied the suggested fix?

```javascript
module('Integration | Component | rental', function (hooks) {
  setupRenderingTest(hooks);

  /* NEW */
  hooks.beforeEach(function () {
    this.owner.setupRouter();
  });
```

**Comment by fdrobidoux** [comment]
Yes, this is after adding the fix.

**Comment by ijlee2** [comment]
@fdrobidoux Okay, thank you for confirming. I'll have a look at what's going on today/tomorrow.

**Comment by ijlee2** [comment]
Yep, I was able to reproduce the issue in an Ember 3.25.1 app. https://github.com/ijlee2/super-rentals-tutorial-3.25

The tutorial is "alive," in the sense that every code presented has been checked through automated testing. (It's a part of our continuous integration.)

As a result, I'm suspecting the error hasn't been caught because the tutorial currently uses Ember 3.24.0 (the version can be seen in https://guides.emberjs.com/release/tutorial/part-1/orientation/).

I'll notify other teams to let them know there may be a regression in 3.25. I appreciate both of you letting us know of the failing test, and hope that it didn't affect your first learning experience of Ember too badly.

If you have additional questions about the tutorial, or Ember.js, please feel free to reach out on [Ember Discord](https://discord.gg/emberjs). The Learning Team can be reached through the Discord channel called `#dev-ember-learning`. 🧡

**Comment by ijlee2** [comment]
Related issue in `ember-source`: https://github.com/emberjs/ember.js/issues/19408
